ts-redis/test/pulumi/index.ts

```diff
@@ -0,0 +1,26 @@
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi"
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const stackPolicy: StackValidationPolicy = {
```

Comment:
more of a consistency comment but this one's missing checking for the lambda.

Also in some we seem to do StackPolicy and check for everything in the stack, and in others we have one policy for each infra type, (lambda, ecs, ..)

Reply:
agreed, i want to write a helper that reads the klotho config and then does the checks based on that so hopefully this is temporary